docs/src/main/sphinx/connector/delta-lake.rst

```diff
@@ -0,0 +1,603 @@
+* Network access from the coordinator and workers to the Delta Lake storage.
+* Access to the Hive metastore service (HMS) of Delta Lake or a separate HMS.
+* Network access to the HMS from the coordinator and workers. Port 9083 is the
+  default port for the Thrift protocol used by the HMS.
+
+Configuration
+-------------
+
+The connector requires a Hive metastore for table metadata and supports the same
+metastore configuration properties as the :doc:`Hive connector
+</connector/hive>`. At a minimum, ``hive.metastore.uri`` must be configured.
+
+The connector recognizes Delta tables created in the metastore by the Databricks
+runtime. If non-Delta tables are present in the metastore, as well, they will
+not be visible to the connector.
+
+To configure the Delta Lake connector, create a catalog file, for example
+``etc/catalog/delta.properties``, that references the ``delta-lake`` connector.
+Update the ``hive.metastore.uri`` with the URI of your Hive metastore Thrift
+service:
+
+.. code-block:: properties
+
+    connector.name=delta-lake
+    hive.metastore.uri=thrift://example.net:9083
+
+If you are using AWS Glue as Hive metastore, you can simply set the metastore to
+``glue``:
+
+.. code-block:: properties
+
+    connector.name=delta-lake
+    hive.metastore=glue
+
+The Delta Lake connector reuses certain functionalities from the Hive connector,
+including the metastore thrift and glue configuration, detailed in the
+:doc:`Hive connector documentation </connector/hive>`.
+
+To configure access to S3 and S3-compatible storage, Azure storage, and others,
+consult the Amazon S3 section of :doc:`Hive connector documentation
+</connector/hive>` or :doc:`the Azure storage documentation
+</connector/hive-azure>`, respectively.
+
+Configuration properties
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+The following configuration properties are all using reasonable, tested default
+values. Typical usage does not require you to configure them.
+
+.. list-table:: Delta Lake configuration properties
+    :widths: 30, 50, 20
+    :header-rows: 1
+
+    * - Property name
+      - Description
+      - Default value
+    * - ``delta.metadata.cache-ttl``
+      - Frequency of checks for metadata updates, equivalent to transactions, to
+        update the metadata cache
+      - 5 min
+    * - ``delta.metadata.live-files.cache-size``
+      - Amount of memory allocated for caching information about files. Needs
+        to be specified in data size values such as ``64 MB``.
+      - Calculated to 10% of the maximum memory allocated to the coordinator
+        JVM.
+    * - ``delta.max-partitions-per-writer``
+      - Maximum number of partitions per writer.
+      - 100
+    * - ``delta.hide-non-delta-lake-tables``
+      - Hide information about tables that are not managed by Delta Lake. Hiding
+        only applies to tables with the metadata managed in a Glue catalog, does
+        not apply to usage with a Hive metastore service.
+      - false
```

Comment:
code hightlight

Reply:
Do numerical entries get code highlights, as well? As of now, only some are already highlighted in the tables of this document.